packages/sdks/e2e/qwik/package.json

```diff
@@ -9,8 +9,8 @@
     "build.client": "vite build",
     "build.ssr": "vite build --ssr src/entry.express.tsx",
     "dev": "vite",
-    "dev.debug": "node --inspect-brk node_modules/vite/bin/vite.js --mode ssr",
-    "dev.ssr": "node --inspect node_modules/vite/bin/vite.js --mode ssr",
+    "dev.debug": "vite --mode ssr",
+    "dev.ssr": "vite --mode ssr",
```

Comment:
@mhevery `vite` is now grabbed from the root of SDKs so this path no longer works, had to make this change to get the PR working again